proca/lib/proca/target_email.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Proca.TargetEmail do
+  @moduledoc """
+  TargetEmail contains the email data for a Target
+  """
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "target_emails" do
```

Comment:
I don't get splitting target and target email, the use cases I can imaging having something different than a 1-1:
- keeping a log (but you'd need a status active/inactive)
- have alternate emails (needs status/role too)

is this possible to have more than one email per target?

Reply:
We talked on signal